ods_ci/tests/Tests/500__jupyterhub/minimal-vscode-test.robot

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
+Documentation    Test Suite for Visual Studio Code (VSCode) image
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/OCPDashboard/Builds/Builds.robot
+Library          Screenshot
+Library          DebugLibrary
+Library          JupyterLibrary
+Suite Setup      Verify VSCode Image Suite Setup
+Suite Teardown   End Web Test
+Test Tags       JupyterHub
+
+
+*** Variables ***
+${NOTEBOOK_IMAGE} =         code-server
+
+
+*** Test Cases ***
+Verify VSCode Image Can Be Spawned
+    [Documentation]    Spawns vscode image
+    [Tags]  Sanity    Tier1
+    Pass Execution    Passing tests, as suite setup ensures that image can be spawned
+
+
+*** Keywords ***
+Verify VSCode Image Suite Setup
+    [Documentation]    Suite Setup, spawns vscode image
+    Begin Web Test
+    Launch JupyterHub Spawner From Dashboard
+    Spawn Notebook With Arguments  image=${NOTEBOOK_IMAGE}  size=Small
+
+Close Previous Server
+    [Documentation]  Closes previous server and goes back to Spawner
+    Clean Up Server
+    Stop JupyterLab Notebook Server
```

Comment:
does it work for VS Code as well?

Reply:
yeah, no... it isn't being called at all, actually :slightly_frowning_face: let me check

Reply:
Updated - I removed this part completely and changed the teardown method at the beginning of the file that stops the created workbench at the end of the test execution.